ovn-tester/ovn_workload.py

```diff
@@ -61,17 +61,27 @@ def __init__(self, phys_node, container, mgmt_net, mgmt_ip):
 
 class CentralNode(Node):
     def __init__(
-        self, phys_node, db_containers, relay_containers, mgmt_net, mgmt_ip
+        self,
+        phys_node,
+        db_containers,
+        relay_containers,
+        mgmt_net,
+        mgmt_ip,
+        gw_net,
+        idx,
```

Comment:
Maybe this can be a more explicit name, e.g. "az_index" or even just "az".

Reply:
@dceara ack, will do